mne/time_frequency/tests/test_tfr.py

```diff
@@ -133,11 +133,22 @@ def test_time_frequency():
     Fs = raw.info['sfreq']  # sampling in Hz
     tfr = cwt_morlet(data[0], Fs, freqs, use_fft=True, n_cycles=2)
     assert_true(tfr.shape == (len(picks), len(freqs), len(times)))
+    tfr2 = cwt_morlet(data[0], Fs, freqs, use_fft=True, n_cycles=2,
+                      time_range=(0, 2))
+    assert_true(tfr2.shape == (len(picks), len(freqs), 2))
 
     single_power = single_trial_power(data, Fs, freqs, use_fft=False,
                                       n_cycles=2)
-
-    assert_array_almost_equal(np.mean(single_power), power.data)
+    single_power2 = single_trial_power(data, Fs, freqs, use_fft=False,
+                                      n_cycles=2, time_range=(0, 2))
```

Comment:
I agree, but since it's a bit of more work, I went for the shortest path for the first try.
Or it could be renamed something like "time_ind_range"...